### DISCUSSION TITLE
I keep getting denied from the student developer pack

### DISCUSSION THREAD

**Discussion by dxkyy** [community/community#142068] [GitHub Education]
### Hi everyone,

My school provides a two-sided student ID. On the front, it displays information from the year I first received the card, and on the back, there are stamps for each year I've extended its validity. However, the year numbers on the back are difficult to read due to the overlapping stamps.

For the past two years, I successfully applied to the GitHub Student Developer Program, but this year my application keeps getting denied with the message: "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted."

I've followed all the instructions in the email, and even when I submit a photocopy of my student ID showing both sides, my application is still rejected. What can I do to resolve this?

**Comment by dxkyy** [comment]
I just sent a new application with both sides of my school id but I got rejected immediately after submitting my application

**Comment by dxkyy** [comment]
I just sent a new application with another form of proof (which is dated) and I got this response when I got rejected:
"Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.". It literally is dated. What is this??

**Reply by lannahirave** [reply]
I guess their OCR broke, same issue.
NVM, had to actually set in billings my name in native language, and got approved.
